BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-138: Create Admin necessary screens

- Make Admin home (Project table)
- Create, edit, delete, Project
- Assign employee to projects
- Add employee Table
- Edit Employee
- Add Admin profile

QA: 
To test this PR logged as ADMIN should render projects table in home, Employees table in employees view and your profile in profile view! 
As ADMIN, you can create, edit and delete projects, edit employees, edit your profile!
